assets/src/components/resource/editors/ActivityBankSelectionEditor.tsx

```diff
@@ -134,3 +134,22 @@ export const ActivityBankSelectionEditor = (props: ActivityBankSelectionEditorPr
     </div>
   );
 };
+
```

Comment:
I refactored all editor outline components to be part of their corresponding editor file for better organization and colocation of similar logic